leval/evaluator.py

```diff
@@ -7,14 +7,18 @@
     InvalidConstant,
     InvalidNode,
     InvalidOperation,
+    NoSuchValue,
     Timeout,
     TooComplex,
 )
 from leval.universe.base import BaseEvaluationUniverse
 from leval.utils import expand_name
 
+NoneType = type(None)
```

Comment:
Prefer `types.NoneType` when it is importable:

```suggestion
try:
    from types import NoneType
except ImportError:  # Python <3.10
    NoneType = type(None)
```